pkg/k8s/network_policy.go

```diff
@@ -82,14 +86,17 @@ func ParseNetworkPolicy(np *networkingv1.NetworkPolicy) (api.Rules, error) {
 		ingress := api.IngressRule{}
 		if iRule.From != nil && len(iRule.From) > 0 {
 			for _, rule := range iRule.From {
-				endpointSelector, err := parseNetworkPolicyPeer(namespace, &rule)
-				if err != nil {
-					return nil, err
+				endpointSelector := parseNetworkPolicyPeer(namespace, &rule)
+
+				// Case where no label-based selectors were in rule.
+				if endpointSelector != nil {
```

Comment:
Please add a debug message that we are skipping to import a rule here

Reply:
I'll add a log saying that the rule has no PodSelector / NamespaceSelector in `ParseNetworkPolicyPeer`. We aren't skipping to import a rule because the rule could also have `IPBlock` restrictions in it which are parsed following this line. Is that OK?

Reply:
Ah, the comment is incorrect, I'll move it accordingly as well.